src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lt1 = new Lottery();
+        Lottery lt2 = new Lottery();
+        Lottery lt3 = new Lottery();
+        Lottery[] lotteries = new Lottery[] { lt1,lt2,lt3 };
+        for (Lottery lottery : lotteries) {
+            System.out.println(lottery.getRandomBall());
+        }
```

Comment:
You just need one lottery instance, use for-loop (instead of for-each) with bound 3 if you want to run test three times. Then in for loop just call `getRandomBall`. It's designed to return random ball after all for the same one instance :)